e2e/images/redis/producer/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM abhirockzz/redis-streams-producer@sha256:79a6dda04b1a54c4ddd1e802ac8447b9c802adf9860cba8cf65017a3e333e7fa
```

Comment:
We could ask @abhirockzz for the sources of this image

Reply:
sure.. will do.